docusaurus.config.js

```diff
@@ -48,6 +51,12 @@ const config = {
           src: 'images/botkube.png',
         },
         items: [
+          {
+            type: 'doc',
+            docId: 'installation/installation',
+            position: 'left',
+            label: 'Docs',
```

Comment:
```suggestion
            label: 'Documentation',
```
Personally, I like if we don't use the short form, but it's up to you.